include/swift/AST/ASTMangler.h

```diff
@@ -167,7 +167,7 @@ class ASTMangler : public Mangler {
   // - Associated function kind.
```

Comment:
Linear map kind.

Reply:
Done in https://github.com/apple/swift/pull/26389/commits/724aa0cf67c5902d95b8ef8b362ece8cba96b664.